src/rgw/rgw_usage.cc

```diff
@@ -146,15 +146,8 @@ int RGWUsage::show(const DoutPrefixProvider *dpp, rgw::sal::Store* store,
     formatter->close_section(); // summary
   }
   
-  // Added storage stats for a user as part of this API
```

Comment:
why this code removed ? for plain ceph this function call is required

Reply:
My bad added again. 👍🏼